Nodejs/Setup/swix/NodejsTools_sideload.vsmanproj

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
should add sideloading script to build script. also remove TS dependencies from sideloading